spec/controllers/strategies_controller_spec.rb

```diff
@@ -90,7 +90,7 @@
       context 'when the comment is saved' do
         it 'responds with an OK status' do
           post :comment, params: valid_comment_params
-          expect(response.status).to eq(200)
+          expect(response).to be_successful
```

Comment:
I think we should go with the specific status here. Either 204 or 200

Reply:
I see your point, changed it back to check the expected status code. Thanks for the feedback